website/docs/pushdown.md

```diff
@@ -52,3 +52,28 @@ Numbers outside the range of the safe IEEE 754 precision (`< -9007199254740991.0
 will prefetch all numbers larger/smaller than max/min value of the range.
 
 <!-- markdownlint-restore -->
+
+## Supported pushdown on `find` command arguments
+
+The following table shows supported command arguments for the PostgreSQL backend.
+It applies `LIMIT` clause for `limit()` and `ORDER BY` clause for `sort()` on SQL queries.
+
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable MD001 MD033 MD051 -->
+
+| `find` command arguments | Supported               |
+| ------------------------ | ----------------------- |
+| `limit()`                | ✅                      |
+| `limit().skip()`         | ✖️                      |
```

Comment:
But those are not `find` command arguments.

Here is the `find` command documentation with arguments: https://www.mongodb.com/docs/manual/reference/command/find/

Here is `mongosh` method documentation with cursor modififers: https://www.mongodb.com/docs/manual/reference/method/db.collection.find/

I strongly believe we should document the former, not the latter, and not a mix of both. Just like https://docs.ferretdb.io/reference/supported-commands/ documents commands, not NodeJS or mongosh methods.

Reply:
Oops, my mistake I've now added arguments similarly to our documentation. The combination of arguments that behaves differently is a bit awkward in the table. Let me know what you think :hugs: